drivers/hmis/app/models/hmis/hud/validators/enrollment_validator.rb

```diff
@@ -30,23 +30,50 @@ def self.duplicate_member_full_message
   end
 
   def self.already_enrolled_full_message
-    'Client is already enrolled in this project.'
+    'Client has another enrollment in this project that conflicts with this entry date.'
+  end
+
+  def self.find_conflict_severity(enrollment)
+    conflict_scope = Hmis::Hud::Enrollment
+      .enrollments_including_wip
```

Comment:
This may have been added erroneously? I think it can be fully removed. Its not a scope on enorllment.